docs/developer-docs/multi-chain/chain-key-tokens/cketh/making-transactions.mdx

```diff
@@ -95,6 +90,8 @@ dfx canister --network ic call minter withdraw_eth "(record {amount = SMALL_AMOU
 
 This withdraw is done in wei, the smallest denomination of ETH. You can use [this converter](https://eth-converter.com/) to convert ETH to wei.
 
+The first step uses `LARGE_AMOUNT_WEI` while the second step uses `SMALL_AMOUNT_WEI`. This is because once you approve an amount, which may be large, the second command can be run multiple times with smaller amounts until the entire allowance is used.
+
 For example, to withdraw 1 ETH:
 
 ```
```

Comment:
The command below should be

```
dfx canister --network ic call minter withdraw_eth "(record {amount = 1_000_000_000_000_000_000; recipient = \"YOUR_ETH_ADDRESS\"})"
```

You may also want to use a sample destination address, e.g., `0xAB586458E47f3e9D350e476fB7E294a57825A3f4` (i.e., `recipient = \"0xAB586458E47f3e9D350e476fB7E294a57825A3f4\"`).